src/Sai2Model.cpp

```diff
@@ -1034,6 +1034,48 @@ GraspMatrixData Sai2Model::environmentalGraspMatrixAtGeometricCenter(
 	return G_data;
 }
 
+void Sai2Model::Sai2Model::forwardDynamics(VectorXd& ddq, const VectorXd& tau) {
+	ForwardDynamics(*_rbdl_model, _q, _dq, tau, ddq);
+}
+
+Vector6d Sai2Model::Sai2Model::jDotQDot(const string& link_name, const Vector3d& pos_in_link, const bool update_kinematics) {
+	Vector3d prev_gravity = _rbdl_model->gravity;
+	_rbdl_model->gravity.setZero();
+	Vector6d acc6d = CalcPointAcceleration6D(*_rbdl_model, _q, _dq, VectorXd::Zero(_ddq.size()), linkIdRbdl(link_name), pos_in_link, true);
+	acc6d.head(3).swap(acc6d.tail(3));
+	_rbdl_model->gravity = prev_gravity;
+	updateKinematics();
+	return acc6d;
+}
+
+void Sai2Model::Sai2Model::addLoad(const std::string link_name,
```

Comment:
Don't include those add load/remove load functions and the associated example in this PR. Make another one dedicated to adding/removing loads

Reply:
Will move this to another branch